src/nitclk/metadata/functions.py

```diff
@@ -134,11 +134,14 @@
             },
             {
                 'direction': 'in',
+                'default_value': 'datetime.timedelta(seconds=0.0)',
+                'python_api_converter_name': 'convert_timedelta_to_seconds',
+                'type_in_documentation': 'float in seconds or datetime.timedelta',
+                'type': 'ViReal64',
```

Comment:
Please confirm these changes were made in internal NI metadata.

Reply:
They will be soon